### PR TITLE
Adds "Merged" and "Closed" stamps

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Adds "Merged" and "Closed" stamps, along with fluff paper containing a random amount of randomly generated pull requests. They can be found in a briefcase in the RD's locker, or rarely in maint.

image

## Why It's Good For The Game

That damn griefer george melons came to me in a dream and asked me to make funny stamps. They can be found in the research director's locker, and also in maint.

## Changelog

:cl: BurgerBB
add: Adds "Merged" and "Closed" stamps. They can be found in the research director's locker.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/paperwork/stamps.dm [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+/obj/item/stamp/coderbus/merged
+	name = "merged rubber stamp"
+	icon_state = "stamp-merged"
+	dye_color = DYE_PURPLE
+
+/obj/item/stamp/coderbus/closed
+	name = "closed rubber stamp"
+	icon_state = "stamp-closed"
+	dye_color = DYE_RED
```
Comment (theselfish) [inline modular_zubbers/code/modules/paperwork/stamps.dm:13]:
```suggestion
	dye_color = DYE_RED
	
```
Trailing space moment.

File: modular_zubbers/code/modules/paperwork/paper_premade.dm [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	<h1>Changelog</h1>
+	<hr>
+	CL [author]<br>
+	[pick("del","remove","balance","fix","bugfix")]: Removes [thing_to_remove]<br>
+	/CL
+	"}
+
+	name = "[initial(name)] - pull request #[pr_number], [title]"
+
+	return ..()
```
Comment (theselfish) [inline modular_zubbers/code/modules/paperwork/paper_premade.dm:52]: Actually wait you need to hit enter on like. All of these.

File: modular_zubbers/code/modules/paperwork/folders_premade.dm [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/obj/item/folder/coderbus
+	icon_state = "folder_syndie" //honestly
+
+/obj/item/folder/coderbus/Initialize(mapload)
+	. = ..()
+	for(var/i in 1 to rand(3,8))
+		new /obj/item/paper/fluff/merge_my_fucking_pr(src)
+	update_appearance()
```
Comment (theselfish) [inline modular_zubbers/code/modules/paperwork/folders_premade.dm:8]: This one too!

**Comment by ReturnToZender** [comment]
Why is that damn griefer George Melons always a command member as an antag and why does it say "I demanding" and not "I'm demanding"

**Comment by BurgerLUA** [comment]
> Why is that damn griefer George Melons always a command member as an antag and why does it say "I demanding" and not "I'm demanding"

that later thing was fixed after I uploaded the screenshot and I was too lazy to update it

File: modular_zubbers/code/modules/paperwork/stamps.dm [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/obj/item/stamp/coderbus
+	icon = 'modular_zubbers/icons/obj/service/bureaucracy.dmi'
+	icon_state = "null"
```
[the remark this answers is not in the document]
Comment (BurgerLUA) [reply modular_zubbers/code/modules/paperwork/stamps.dm]: yes, I changed to the actual value of null just now

**Comment by StrangeWeirdKitten** [comment]
I just saw that, all good.

**Pull request merged**